Add plugin: Text Focus

usysrc opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

Link to my plugin: [https://github.com/usysrc/obsidian-text-focus-plugin](https://github.com/usysrc/obsidian-text-focus-plugin)

## Release Checklist
- [x] I have tested the plugin on
  - [ ]  Windows
  - [x]  macOS
  - [ ]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the developer policies at https://docs.obsidian.md/Developer+policies, and have assessed my plugins's adherence to these policies.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

joethei commented:
Remove everything from your readme that is not meant for end-users.

["fundingUrl": "",](https://github.com/usysrc/obsidian-text-focus-plugin/blob/c044f877275b2313ed587fbd1f651860a3ce2b69/manifest.json#L9)
This is meant for links to services like _Buy me a coffee_, _GitHub sponsors_ and so on, if you don't have such a link omit this.

[async wait(milliseconds: number): Promise<void> {](https://github.com/usysrc/obsidian-text-focus-plugin/blob/c044f877275b2313ed587fbd1f651860a3ce2b69/main.ts#L43-L45)
The Obsidian API already exports a `sleep` function for this purpose.

usysrc commented:
> Remove everything from your readme that is not meant for end-users.
> 
> ["fundingUrl": "",](https://github.com/usysrc/obsidian-text-focus-plugin/blob/c044f877275b2313ed587fbd1f651860a3ce2b69/manifest.json#L9) This is meant for links to services like _Buy me a coffee_, _GitHub sponsors_ and so on, if you don't have such a link omit this.
> 
> [async wait(milliseconds: number): Promise {](https://github.com/usysrc/obsidian-text-focus-plugin/blob/c044f877275b2313ed587fbd1f651860a3ce2b69/main.ts#L43-L45) The Obsidian API already exports a `sleep` function for this purpose.

Implemented plugin requested changes

- removed everything from the README.md and added simple description and usage
- removed fundingUrl from manifest
- used obsidian API sleep instead of wait
